sormas-api/src/main/java/de/symeda/sormas/api/user/DefaultUserRole.java

```diff
@@ -1626,4 +1662,20 @@ public String toString() {
 	public String toShortString() {
 		return I18nProperties.getEnumCaptionShort(this);
 	}
+
+	public static DefaultUserRole forName(String name) {
```

Comment:
Couldn't find where this is used. If this is needed: Why not use DefaultUserRole.valueOf?